server/migrations/migrations.go

```diff
@@ -0,0 +1,6 @@
+package migrations
+
+import "embed"
+
+//go:embed *.sql
```

Comment:
Makes sense to force here just to embed the files in the migrations folder? Is there a change of embedding files in the `examples` folder by mistake?

Reply:
this is embed during build time. this helps reduce the configurations needed to boot the app. We used to have an internal config for migrations path that was only used for testing, so this change makes the migrations "hardcoded" withinh the app so no configuration is needed